Avoid shell injection in examples (#26137)
github/docs: content/actions/using-workflows/workflow-commands-for-github-actions.md

During the execution of a workflow, the runner generates temporary files that can be used to perform certain actions. The path to these files are exposed via environment variables. You will need to use UTF-8 encoding when writing to these files to ensure proper processing of the commands. Multiple commands can be written to the same file, separated by newlines.

> Most commands in the following examples use double quotes for echoing strings, which will attempt to interpolate characters like `$` for shell variable names. To always use literal values in quoted strings, you can use single quotes instead.

Reviewer: Why have you removed the above? The seems to still apply.

Reviewer: Thanks for the clarification. Let's take this out, as you suggest.

{% warning %}

> **Warning:** Make sure the delimiter you're using cannot occur on its own line in the value. If the value is completely arbitrary then don't use this format at all. Write to a separate file instead.

Reviewer: I think it would be clearer to say:
```suggestion
**Warning:** Make sure the delimiter you're using won't occur on a line of its own within the value. If the value is completely arbitrary then you shouldn't use this format. Write the value to a file instead.
```
